h2o-algos/src/main/java/hex/schemas/CoxPHV3.java

```diff
@@ -50,7 +50,7 @@
     public double lre_min;
 
     @API(help="Maximum number of iterations.", direction = API.Direction.INOUT)
```

Comment:
This is a hyperparameter in other algorithms. Is it gridable in CoxPH? If so, then we need to add "gridable = true"

Reply:
It is conceptually gridable but I don't want to enable grid search for CoxPH just yet. This is untested